extensions-core/stats/src/main/java/org/apache/druid/query/aggregation/variance/StandardDeviationPostAggregator.java

```diff
@@ -150,4 +150,10 @@ public boolean equals(Object o)
 
     return true;
   }
+
+  @Override
+  public int hashCode()
+  {
+    return Objects.hash(getName(), getFieldName(), getEstimator(), isVariancePop);
```

Comment:
Might as well use direct field accesses, to match what `equals` does.

Reply:
Done :+1: